src/api/default-request.js

```diff
@@ -21,6 +21,8 @@ const getRequest = function (path, callback, errorHandler, configuration) {
       callback(data)
     })
     .catch(error => {
+      console.log(path)
```

Comment:
is this expected?

Reply:
No, forgot to remove this after tracking error.
Will remove